{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/config/settings/local.py

```diff
@@ -93,3 +99,8 @@
         "silk_request",
     )
 )
+
+CSP_DEFAULT_SRC = ["'self'", "localhost:5173", "ws://localhost:5173"]
+CSP_IMG_SRC = ["'self'", "images.unsplash.com"]
+CSP_SCRIPT_SRC += ["localhost:5173", "cdn.jsdelivr.net"]
```

Comment:
Oh, one thing I should point out here. Really unsplash and the cdn don't belong in the local config, they should be in the base. But if you're making a real app I'm assuming/hoping you're not going to be using either of those, so I feel a bit safer leaving them out of the base config and just having them here where they can't really cause any harm.

Not really sure, though.